_posts/2017/2017-08-17-tracing-the-suspect.md

```diff
@@ -0,0 +1,245 @@
+---
+layout: post
+title: "Tracing the suspect - a microservices murder mystery"
```

Comment:
Rename file name same as title: 
_posts/2017/2017-08-17-tracing-the-suspect-a-microservices-murder-mystery.md